Test/gsapTests/EventDrivenPrognoserTests.cpp

```diff
@@ -0,0 +1,212 @@
+/**  Event-Driven Prognoser Tests - Header
+ *   @class     EventDrivenPrognoserTests EventDrivenPrognoserTests.h
+ *
+ *   @brief     Event-Driven Prognoser Test Class
+ *
+ *   Tests for event-driven prognoser class.
+ *
+ *   @author    Chris Teubert
+ *   @version   1.2.0
+ *
+ *      Created: December 17, 2018
+ *
+ *   @copyright Copyright (c) 2018 United States Government as represented by
+ *     the Administrator of the National Aeronautics and Space Administration.
+ *     All Rights Reserved.
+ */
+
+#include <queue>
+#include <thread>
```

Comment:
Apparently all three OS's are very forgiving on threading includes (or the right headers are included elsewhere by the headers included below).

In any case, I don't think we need to include `<thread>`, but we do need `<mutex>` and `<condition_variable>`.